quantum/keymap.h

```diff
@@ -17,24 +17,4 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #pragma once
 
-#include <stdint.h>
-#include <stdbool.h>
-#include "platform_deps.h"
-#include "action.h"
-#include "keycode.h"
-#include "report.h"
-#include "host.h"
-#include "debug.h"
-#include "keycode_config.h"
-#include "gpio.h" // for pin_t
-
-#include "quantum_keycodes.h"
-
-#include "keymap_common.h"
-
-#ifdef ENCODER_MAP_ENABLE
-// Ensure we have a forward declaration for the encoder map
-#    include "encoder.h"
-#endif
-
-#include "keymap_introspection.h"
+#pragma message("'keymap.h' should no longer be included!")
```

Comment:
Maybe we do a static assert of false, force failure?

Reply:
Will add to phase 2.